CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
-project(facil.io C)
+project(facilio C)
```

Comment:
Is there a reason to drop the `.`?

If so, I'd like to know.

Reply:
It is confusing (and probably not best-practice) using a `.` for library names.
This character is usually reserved as separator between library name, library suffix and version.